images/macos/helpers/Common.Helpers.psm1

```diff
@@ -166,3 +166,12 @@ function Get-Architecture {
 
     return $arch
 }
+
+Function Test-CommandExists {
+    param
+    (
+        $command
```

Comment:
nit: `[Parameter(Mandatory)] [string] $Command,`
(mandatory attribute, var type and capitalize variable name to be consistent with other functions

Reply:
fixed